apps/gitness/src/pages/pull-request/pull-request-compare-page.tsx

```diff
@@ -0,0 +1,273 @@
+import { useEffect, useMemo, useState } from 'react'
+import { SandboxPullRequestCompare, SkeletonList } from '@harnessio/playground'
+import { useNavigate, useParams } from 'react-router-dom'
+import {
+  CreateRepositoryErrorResponse,
+  useCreatePullReqMutation,
+  OpenapiCreatePullReqRequest,
+  CreatePullReqOkResponse,
+  RepoBranch,
+  useListBranchesQuery,
+  useFindRepositoryQuery,
+  useListCommitsQuery,
+  TypesCommit,
+  useRawDiffQuery,
+  mergeCheck,
+  useDiffStatsQuery
+} from '@harnessio/code-service-client'
+import { PathParams } from '../../RouteDefinitions'
+import { changesInfoAtom, DiffFileEntry, DiffViewerExchangeState, FormFields } from './types/types'
+import { useGetRepoRef } from '../../framework/hooks/useGetRepoPath'
+import { normalizeGitRef } from '../../utils/git-utils'
+import { useAtom } from 'jotai'
+import { compact, isEqual } from 'lodash-es'
+import { parseSpecificDiff } from './diff-utils'
+import { changedFileId, DIFF2HTML_CONFIG, normalizeGitFilePath } from './utils'
+import * as Diff2Html from 'diff2html'
+
+export interface TypesUser {
+  admin?: boolean
+  blocked?: boolean
+  created?: number
+  display_name?: string
+  email?: string
+  uid?: string
+  updated?: number
+}
+
```

Comment:
I think a similar interface is given by code-service-client? Do we need to duplicate it here?

Reply:
didnt need it i realized haha @sans-harness